.gitignore

```diff
@@ -2,4 +2,6 @@ build/
 logs/
 nbproject/
 vendor/
+.idea/
 composer.lock
```

Comment:
We'd better add (as a separate PR) the `composer.lock` to repo to ensure that we're using same dependency versions for development and on Travis.